css/build.css

```
@@ -144,4 +144,8 @@
 /* For native devices */
 .native .fa-times-thin {
   padding-top: 4px;
-} 
\ No newline at end of file
+}
+
+[data-dynamic-lists-id] {
```

```suggestion
.mode-interact [data-dynamic-lists-id] {
```

The `.mode-interact` selector should be added so that the minimum height is only applied to edit mode. This is because we're only making this change to make sure the widget is editable. In preview, if the widget instance is empty, then it shouldn't take up unnecessary space with a minimum height.